src/components/app-configure/general/AppDelete.vue

```diff
@@ -37,8 +37,8 @@ async function handleAppDeletion() {
   loaderStore.showLoader('Deleting App...')
   try {
     const appId = props.appId as number
-    await deleteApp(appId)
-    appsStore.deleteApp(appId)
+    await deleteApp(appId, 'testnet')
+    appsStore.deleteApp(appId, 'testnet')
```

Comment:
Only testnet apps can be deleted?

Reply:
let me check

Reply:
fixed - https://github.com/arcana-network/developer-dashboard/pull/173/commits/c5a39252cca800e19358d39213425ed65901b30c